tools/nni_annotation/examples/mnist_with_annotation.py

```diff
@@ -247,6 +247,7 @@ def generate_defualt_params():
 
 
 if __name__ == '__main__':
+    """@nni.next_parameter()"""
```

Comment:
what is the difference between @nni.next_parameter() and @nni.get_next_parameter()?

Reply:
Thanks, it is a mistake.